src/modules/add.js

```diff
@@ -0,0 +1,49 @@
+const inputForm = document.querySelector('.input-form');
+
+export const tasksHtml = ({ index, description }) => `
+<li class="list" id="${index}" draggable="true" data-id="ee" >
+    <input type="checkbox" name="task" class="check" data-asign="${index}" >
+    <input type="text"  class="input-list" value="${description}">
+    <i class="material-icons dots">
+        more_vert
+    </i>
+    <i class="material-icons delete" >
+        delete
+    </i> 
+</li>
+`;
+
+export const addTask = (arr) => {
+  if (inputForm.value.trim() === '') {
+    return;
+  }
+
+  if (arr.some((task) => task.description === inputForm.value)) {
+    alert('task already added');
+    return;
+  }
+
+  const task = {
+    index: arr.length,
+    description: inputForm.value,
+    completed: false,
+  };
```

Comment:
- [ ] By default,  the property `index` should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Which means that index should start from 1. 
You can simply do it by adding + 1 to array length 
`index: arr.length + 1`